tests/manual-test-cases/Group5-Functional-Tests/5-22-NFS-Volume.robot

```diff
@@ -0,0 +1,249 @@
+    :FOR  ${id}  IN  @{containers}
+    \   ${rc}  ${stopOutput}=  Run And Return Rc And Output  docker %{VCH-PARAMS} stop ${id}
+    \   Should Be Equal As Integers  ${rc}  0
+
+
+Simple docker volume inspect
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} volume inspect ${nfsNamedVolume}
+    Should Be Equal As Integers  ${rc}  0
+    ${output}=  Evaluate  json.loads(r'''${output}''')  json
+    ${id}=  Get From Dictionary  ${output[0]}  Name
+    Should Be Equal As Strings  ${id}  ${nfsNamedVolume}
+
+Simple Volume ls test
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} volume ls
+    Should Be Equal As Integers  ${rc}  0
+    Should Contain  ${output}  vsphere
+    Should Contain  ${output}  ${nfsNamedVolume}
+    Should Contain  ${output}  ${nfsUnNamedVolume}
+    Should Contain  ${output}  DRIVER
+    Should Contain  ${output}  VOLUME NAME
+
+Volume rm tests
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} volume rm ${nfsUnNamedVolume}
+    Should Be Equal As Integers  ${rc}  0
+
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} volume ls
+    Should Be Equal As Integers  ${rc}  0
+    Should Not Contain  ${output}  ${nfsUnNamedVolume}
+
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} volume rm ${nfsNamedVolume}
+    Should Be Equal As Integers  ${rc}  1
+    Should Contain  ${output}  Error response from daemon: volume ${nfsNamedVolume} in use by
+
+Kill NFS Server
+    ${rc}  ${runningContainer}=  Run And Return Rc And Output  docker %{VCH-PARAMS} run -d -v ${nfsNamedVolume}:/mydata ${busybox} sh -c "while true; do echo 'Still here...\n' >> /mydata/test_nfs_kill.txt; sleep 1; done"
+    Should Be Equal As Integers  ${rc}  0
+
+    ${rc}  ${tailOutput}=  Run And Return Rc And Output  docker %{VCH-PARAMS} run -v ${nfsNamedVolume}:/mydata ${busybox} sh -c "tail -5 /mydata/test_nfs_kill.txt"
+    Should Be Equal As Integers  ${rc}  0
+    Should Contain  ${tailOutput}  Still here...
+
+    Kill Nimbus Server  %{NIMBUS_USER}  %{NIMBUS_PASSWORD}  ${NFS}
+
+    ${rc}  ${tailOutput}=  Run And Return Rc And Output  docker %{VCH-PARAMS} run -v ${nfsNamedVolume}:/mydata ${busybox} sh -c "tail -5 /mydata/test_nfs_kill.txt"
+    Should Be Equal As Integers  ${rc}  125
+    Should Contain  ${tailOutput}  Server error from portlayer: unable to wait for process launch status:
+
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} run -v ${nfsNamedVolume}:/mydata ${busybox} sh -c "echo 'Where am I writing to?...\n' >> /mydata/test_nfs_kill.txt"
+    Should Be Equal As Integers  ${rc}  125
+
+    ${rc}  ${lsOutput}=  Run And Return Rc And Output  docker %{VCH-PARAMS} run -v ${nfsNamedVolume}:/mydata ${busybox} sh -c "ls mydata"
+    Should Be Equal As Integers  ${rc}  125
+    Should Contain  ${lsOutput}  Server error from portlayer: unable to wait for process launch status:
```

Comment:
Nice, we need bugs for this though.  That error is unclear to the user, if I got that there is no way I would think to check on my NFS server.